docs/reference_guides/core/costing/costing_libraries.rst

```diff
@@ -0,0 +1,12 @@
+Available Costing Libraries
+===========================
+
+In addition to supporting user-developed, custom costing packages, IDAES provides a set of costing libraries for cost calculations:
+
+.. rubric:: Process Costing Library
+
+* :ref:`Process Costing Library<reference_guides/model_libraries/generic/costing/process_costing:Process Costing Library>`
+
+.. rubric:: Power Plant Costing Library
+
+* :ref:`Power Plant Costing Library<reference_guides/model_libraries/power_generation/costing/power_plant_costing:Power Plant Costing Library>`
```

Comment:
I am not sure if this one should be included yet - I have not seen any PR updating either the code or the docs for this to the new API yet.

Reply:
@MAZamarripa and I are working with the power plant costing methods, tests and documentation on another branch - we aim to open a PR soon.

Reply:
I would leave it to that PR to add this then (to avoid any chance of confusion during the gap between merging the two).